src/bin/zmcontrol

```diff
@@ -71,6 +71,7 @@ my %startorder = (
 	"zimbra"	=> 180,
 	"zimbraAdmin"	=> 190,
 	"zimlet"	=> 200,
+	"imap"		=> 210,
```

Comment:
As with comment on previous file, I'd vote for using "imapd" over just "imap" in here.